### PR TITLE
[14.0][MIG] stock_account_operating_unit

### PR DESCRIPTION
**Pull request by newtratip**
Migrated from https://github.com/OCA/operating-unit/pull/254

@alan196, Could you help review please ?

### CODE REVIEW

File: stock_account_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# © 2019 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline stock_account_operating_unit/__manifest__.py:1]:
```suggestion
# 2019 Copyright ForgeFlow S.L.
```

Can you do this in all copyright headers? Sorry for the nitpcking

**Comment by gdgellatly** [comment]
@newtratip this isn't a straight migration, need to consider svl's as well. I will update comment later with some code pointers.

File: stock_account_operating_unit/tests/test_stock_account_operating_unit.py [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+        return picking
+
+    def _confirm_receive(self, user_id, picking):
+        """
+        Checks the stock availability, validates and process the stock picking.
+        """
+        picking.action_confirm()
+        picking.action_assign()
+        res = picking.with_user(user_id).button_validate()
+        validate_id = res["res_id"]
```
Comment (JuanyDForgeflow) [inline stock_account_operating_unit/tests/test_stock_account_operating_unit.py:177]: Test is failing here please fix:
stock_account_operating_unit/tests/test_stock_account_operating_unit.py", line 177, in _confirm_receive
    validate_id = res["res_id"]
KeyError: 'res_id'

**Comment by AaronHForgeFlow** [comment]
@newtratip do you plan to work on this one? Is it ok with you if I supersede your PR?

**Comment by JuanyDForgeflow** [comment]
The account entry is not showing the correct valuation, unit price of the product is 15$ and transfer was for 10 units

image

But the entry has 15$ as valuation 

image

Also when changing the product cost I get this error:
image


@newtratip  Can you please check?

**Comment by francesco-ooops** [comment]
@newtratip could you rebase?

**Pull request closed without merging**